runtimes/common/Cargo.toml

```diff
@@ -13,6 +13,8 @@ scale-info = {version = "1.0", default-features = false, features = ["derive"]}
 serde = {version = "1.0.124", optional = true, features = ["derive"]}
 smallvec = "1.7.0"
 
+attestation = {default-features = false, path = "../../pallets/attestation"}
```

Comment:
Is this best practice or should we rather define the trait in our support pallet?

Reply:
dependencies can go out from any crate in the runtime folder to any crate in the pallet folder. The other direction does not work. We can't import anything from the runtime folder in the pallet folder since this would introduce a cycle.